### PR TITLE
added padStart and padEnd

### LINKED ISSUE #126: padStart and padEnd
`_.padEnd` / `_.padStart` has corresponding ES6 methods.

### PR DESCRIPTION
**Pull request by bashaus**
Fixes #126 

- [x] Added documentation for `_.padStart` and `_.padEnd` (lodash only)
- [x] Added rules for `_.padStart` and `_.padEnd` (lodash only)
- [x] Added tests for `_.padStart` and `_.padEnd` (lodash only)

### CODE REVIEW

**Comment by cht8687** [comment]
@bashaus Hi, could you resolve the conflicts? so that we can merge this PR. Many thanks for the code.

**Comment by jwallet** [comment]
you are missing a back to the top after browser compatibility and you have a link above your _.pad.. title that should appear in index only

File: README.md [hunk 1]
```diff
@@ -1792,6 +1793,35 @@ Creates an object composed of the object properties predicate returns truthy for
...
+  console.log('123'.padEnd(5, '0'))
+  // output: '12300'
+  ```
+
+#### Browser Support
+
+![Chrome][chrome-image] | ![Firefox][firefox-image] | ![IE][ie-image] | ![Opera][opera-image] | ![Safari][safari-image]
+:-: | :-: | :-: | :-: | :-: |
+  57 ✔  |  48 ✔ |  Not supported  |  44 ✔ |  10 ✔ |
+
```
Comment (jwallet) [inline README.md]: missing back to the top

File: README.md [hunk 2]
```diff
@@ -1792,6 +1793,35 @@ Creates an object composed of the object properties predicate returns truthy for
 
 ## String
 
+[_.padStart and _.padEnd](#_padstart-and-_padend)
+
+### _.padStart and _.padEnd
```
Comment (jwallet) [inline README.md]: this does the actual link in a .md file, you should remove line 1796

File: README.md (hunk 1 above)
Comment (bashaus) [reply README.md]: sorted :)

File: README.md (hunk 2 above)
Comment (bashaus) [reply README.md]: sorted :)

**Comment by bashaus** [comment]
Have resolved merge conflicts and fixed the link issues :)

**Comment by cht8687** [comment]
thanks!

**Pull request merged**